components/src/core/components/Input/switch-input.scss

```diff
@@ -0,0 +1,81 @@
+@import '../../../styles';
+@import 'variables';
+
+.oxd-switch-wrapper {
+  & label {
+    cursor: pointer;
+    display: flex;
+    align-content: center;
+    font-family: $oxd-input-control-font-family;
+    color: $oxd-input-control-font-color;
+    font-size: $oxd-input-control-font-size;
+    font-weight: $oxd-input-control-font-weight;
+    width: fit-content;
+    & input[type='checkbox'] {
+      opacity: 0;
+      position: absolute;
+      + .oxd-switch-input {
+        background-color: $oxd-switch-off-background-color;
+      }
+
+      &:checked + .oxd-switch-input {
+        background-color: $oxd-switch-on-background-color;
+        &::after {
+          transform: translate($oxd-switch-height, -50%);
+        }
+      }
+      &:checked:disabled + .oxd-switch-input {
+        background-color: $oxd-switch-on-disabled-background-color;
+        &::after {
+          transform: translate($oxd-switch-height, -50%);
+        }
+      }
+      &:disabled + .oxd-switch-input {
```

Comment:
can't we add a black line between selectors

Reply:
done